tampermonkey/PPT_Returns_AutoComplete.js

```diff
@@ -292,6 +319,42 @@ const confirmationPage = () => {
      }
 }
 
+/* ####################### Amend submitted returns pages */
+
+const submittedReturns = () => {
+     if (currentPageIs('/plastic-packaging-tax/submitted-returns')) {
+         document.getElementsByClassName('govuk-body')[1].click()
+     }
+}
+
+const existingReturn = () => {
+     if (currentPageIs('/plastic-packaging-tax/viewReturnSummary/22C2')) {
```

Comment:
The `periodKey` might need to be updated to be a wildcard as the value is determined by the stub, so can be different depending on the user.